content/en/Integrations/Azure DevOps/_index.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "Integrate Azure DevOps with Cobalt"
+linkTitle: "Azure DevOps"
+weight: 20
+description: >
+  Push Cobalt findings as work items to Azure DevOps.
+---
+
+{{% pageinfo %}}
+Integrate with Azure DevOps to create work items for your Cobalt findings and streamline your remediation workflows. The availability of this feature depends on your [PtaaS tier](/platform-deep-dive/credits/ptaas-tiers/).
+{{% /pageinfo %}}
+
+## Overview
+
+### How it Works
+
+- Connect your Azure DevOps system to Cobalt and push pentest findings as work items to your team boards.
+- Configure Azure DevOps for each pentest, selecting the Team Project, Work Item Type, and other field mappings for tickets created from findings.
+- When a new finding in the Pending Fix [state](/platform-deep-dive/pentests/findings/finding-states) is reported, a new work item is automatically created in Azure DevOps.
+
+### Limitations
+
+Please be aware of the following considerations when using this integration. We are working to improve the integration and address these limitations.
+
+- Work items will automatically be created when a finding is moved to Pending Fix state. It is not possible to manually create a work item for a finding.
+- Only one Azure DevOps instance can be connected to a Cobalt Organization at a time.
+- Work items created will not appear in the **External Issue References** section of the Cobalt findings page.
+
+## Frequently Asked Questions
+
+Click <i style="font-size:x-large; color: #0047AB" class="fas fa-chevron-right"></i> to view answers.
+
+{{%expand "What is a work item?" %}}
+<br>
+
+In Azure DevOps, a work item is any item that needs to be investigated within a project. Work items may also be referred to as tickets.
+
+In the context of Cobalt integrations, a work item is a finding that was synchronized with Azure DevOps.
+
+{{% /expand %}}
+<br>
+{{%expand "Which fields are required to create Azure DevOps work items?" %}}
+<br>
+
+The following fields are required to create a work item:
+- Project
+- Work item type
+- Title
+- State
+
+Depending on your Azure DevOps process template, additional fields in your work item form may be required. Make sure to provide a value for all required (*) fields. See: [How to set work item field values](/integrations/azure-devops/push-findings/#set-work-item-field-values). 
+
+{{% /expand %}}
+<br>
+{{%expand "Does the integration support custom required fields?" %}}
+<br>
+
+Yes. When configuring the integration for a pentest, you can set values for all standard and custom fields in the work item type template for the Project and Work Item Type selected. See: [How to set work item field values](/integrations/azure-devops/push-findings/#set-work-item-field-values).
+
+{{% /expand %}}
+<br>
+{{%expand "Can I manually create a work item for an individual finding?" %}}
+<br>
+
+No, auto-push is currently not supported. 
```

Comment:
Should this be manual-push? Auto-push is the only method we support.

Reply:
Good catch.  Yes this should read "No, only auto-push is currently supported".

Reply:
Sorry that was a misinformed edit on my part.